presto-main/src/main/java/com/facebook/presto/execution/scheduler/NodeSchedulerConfig.java

```diff
@@ -34,6 +37,7 @@
     private boolean includeCoordinator = true;
     private int maxSplitsPerNode = 100;
     private int maxPendingSplitsPerTask = 10;
+    private NodeSelectionHashFunction nodeSelectionHashFunction = NAIVE;
```

Comment:
remove this

Reply:
I'm more leaning towards having another config to decide between different soft affinity algorithms(it's possible that we add more according to different use cases)  instead of adding another scheduling strategy